components/create_comment/create_comment.tsx

```diff
@@ -378,6 +372,10 @@ class CreateComment extends React.PureComponent<Props, State> {
             }
             this.doInitialScrollToBottom = false;
         }
+
+        if (this.props.createPostErrorId === 'api.post.create_post.root_id.app_error') {
```

Comment:
You'll probably want to also check here that `createPostErrorId` changed to avoid possibly calling `openModal` multiple times if the `CreateComment` renders again for any reason

Reply:
yes i added another check for checking that